src/liballoc/string.rs

```diff
@@ -1567,9 +1534,6 @@ impl String {
     /// Panics if the starting point or end point do not lie on a [`char`]
     /// boundary, or if they're out of bounds.
     ///
-    /// [`char`]: ../../std/primitive.char.html
-    /// [`Vec::splice`]: ../../std/vec/struct.Vec.html#method.splice
```

Comment:
nit: before you linked to core::vec::Vec, but now this uses the imported type (implicitly, through [`Vec::splice`] in the text). Can you use them consistently?

Reply:
It doesn't matter? They'll link to the same thing

Reply:
Ok, nbd then.